README.rst

```diff
@@ -35,6 +36,15 @@ Documentation
 You can read more about `installing Flocker`_, follow a `tutorial`_ and learn about the `features of Flocker and its architecture`_ in the docs.
 
 
+Support
+-------
+
+If you need support to use Flocker, please check the `Flocker Support page`_.
+
+If you have any feature requests or suggestions, we would love to hear about them.
+Please add your ideas to our `UserVoice`_ forum, or file a `GitHub issue`_.
```

Comment:
I don't think that it is useful to call this "UserVoice" - that is an implementation detail and not mentioned in the linked-to page.